core/src/main/resources/hudson/model/ComputerSet/index.jelly

```diff
@@ -37,13 +37,13 @@ THE SOFTWARE.
       <div class="jenkins-app-bar__content">
         <h1>${%Manage nodes and clouds}</h1>
       </div>
-      <l:hasAdministerOrManage>
+      <j:if test="${h.hasAnyPermission(it, it.EXTENDED_READ, app.MANAGE)}">
```

Comment:
This code does not work. The `it` in this case is `ComputerSet`. It is not a `AccessControlled` as requested by `Functions#hasAnyPermission` and, the `it.EXTENDED_READ` has no value because it's `Computer.EXTENDED_READ`.

Due to this, I am getting
> 2022-04-20 12:01:29.506+0000 [id=32]    WARNING h.ExpressionFactory2$JexlExpression#evaluate
: Caught exception evaluating: h.hasAnyPermission(it, it.EXTENDED_READ, app.MANAGE) in /jenk
ins/computer/. Reason: java.lang.IllegalArgumentException
java.lang.IllegalArgumentException

when testing manually this PR.

Reply:
@Wadeck I've made a change, however, I'm not sure if the syntax is correct. Could you please check?

Reply:
@hazzik I would recommend you to test your code by running it locally.

I think your second attempt will not work and provide information in the log. There is no way to pass reference to class like your `Computer` inside Jelly without calling some particular tags before.

Reply:
Thanks @timja.